tests/SampleApps/DotNetCore/Blazor_Function_Sample/blazor-sample-app/Dockerfile

```diff
@@ -5,23 +5,23 @@ COPY . ./
 WORKDIR /app/
 RUN dotnet publish -c Release -o published_app
 
-FROM mcr.microsoft.com/mirror/docker/library/nginx:1.21 AS build
+FROM nginx:1.24.0 AS build
```

Comment:
Any reason we can't bump the version in the tag of the MCR repository to `1.24`? It looks like it's available in the [list of MCR tags](https://[mcr.microsoft.com/v2/mirror/docker/library/nginx/tags/list).

If we move away from MCR, I think (depending on if Sample Apps are scanned) the security portion of the pipeline will fail because we're no longer allowed to use non-MCR/ACR bases within a Dockerfile without an explicit opt-out (which we have in some of our own Dockerfiles, but using the MCR version of this library would be optimal)

Reply:
Works with `mcr.microsoft.com/mirror/docker/library/nginx:1.24`. Thanks